src/components/select/select.template.tsx

```diff
@@ -88,7 +88,8 @@ const SelectValue: FunctionalComponent<
         'mdc-floating-label--float-above':
             (!props.hasEmptyText && props.hasValue) ||
             props.isOpen ||
-            props.readonly,
+            props.readonly ||
+            props.multiple,
```

Comment:
this this will always push the label up 
<img width="731" alt="image" src="https://user-images.githubusercontent.com/35954987/207589927-db5e096b-6780-4273-b926-756c31801239.png">

Reply:
right! added a fixup!

Reply:
https://user-images.githubusercontent.com/50618208/207591670-564455e9-c489-44b6-b404-5b0bdc1ad846.mov